libcst/codemod/commands/tests/test_rename.py

```diff
@@ -0,0 +1,290 @@
+            new_object="B",
+        )
+
+    def test_rename_local_variable(self) -> None:
+        before = """
+            x = 5
+            y = 5 + x
+        """
+        after = """
+            z = 5
+            y = 5 + z
+        """
+
+        self.assertCodemod(
+            before, after, orig_object="x", new_object="z",
+        )
+
+    def test_module_does_not_change(self) -> None:
+        before = """
+            from a import b
+
+            class Foo(b):
+                pass
+        """
+        after = """
+            from a import c
+
+            class Foo(c):
+                pass
+        """
+        self.assertCodemod(
+            before, after, orig_module="a", orig_object="b", new_object="c"
+        )
+
+    def test_other_imports_untouched(self) -> None:
+        before = """
+            import a, b, c
+
+            class Foo(a.z):
+                bar: b.bar
+                baz: c.baz
+        """
+        after = """
+            import b, c
+            import d
+
+            class Foo(d.z):
+                bar: b.bar
+                baz: c.baz
+        """
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="a",
+            orig_object="z",
+            new_module="d",
+            new_object="z",
+        )
+
+    def test_other_import_froms_untouched(self) -> None:
+        before = """
+            from a import b, c, d
+
+            class Foo(b):
+                bar: c.bar
+                baz: d.baz
+        """
+        after = """
+            from a import c, d
+            from f import b
+
+            class Foo(b):
+                bar: c.bar
+                baz: d.baz
+        """
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="a",
+            orig_object="b",
+            new_module="f",
+            new_object="b",
+        )
+
+    def test_no_removal_of_import_in_use(self) -> None:
+        before = """
+            import a
+
+            class Foo(a.b):
+                pass
+            class Foo2(a.c):
+                pass
+        """
+        after = """
+            import a
+            import z
+
+            class Foo(z.b):
+                pass
+            class Foo2(a.c):
+                pass
+        """
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="a",
+            orig_object="b",
+            new_module="z",
+            new_object="b",
+        )
+
+    def test_no_removal_of_import_from_in_use(self) -> None:
+        before = """
+            from a import m
+
+            class Foo(m.CstNode):
+                bar: m.Attribute
+        """
+        after = """
+            from a import m
+            import blah
+
+            class Foo(blah.CstNode):
+                bar: m.Attribute
+        """
+        self.assertCodemod(
+            before,
+            after,
+            orig_module="a",
```

Comment:
Does this work if `orig_module="a.m"` and `orig_object="CstNode"`?

Reply:
Just added another test: https://github.com/Instagram/LibCST/pull/308/files#diff-511afb8b206c354e493c782e1c50abadR268